portal/get_patient_info.php

```diff
@@ -27,6 +27,11 @@
 $landingpage = "index.php?site=" . urlencode($_SESSION['site_id'] ?? ($_GET['site'] ?? 'default'));
 //
 
+if (isset($_REQUEST['redirect'])) {
```

Comment:
is_set can be set but empty, This is one of the reasons PHP created the ?? operator to get rid of the  pain in the #@! is_set && !empty() exal...

Reply:
An alternative here is to use Symfony's `HttpFoundation\Request`

```php
$request = HttpFoundation\Request::createFromGlobals();
$redirect = $request->get('redirect_url_parameter', 'some_default_value');
```

Reply:
I'm glad you remined! I'll get rid of guzzle in my module...

Reply:
The challenge here is that globals.inc.php doesn't get included until later in the script and blows up if there isn't any authorized session so I don't have access to the namespace loader (I know I tried getting it to work on an earlier iteration :( )  I like the suggestion @robertdown but I think I'll just stick with the !empty piece as I don't really want to get into refactoring/messing with the globals.inc.php.